<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ModificarContabilidadBase</class>
 <widget class="QDialog" name="ModificarContabilidadBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>608</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="maximumSize">
   <size>
    <width>2800</width>
    <height>2500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Contabilidad</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/images/iconbulmacont.svg</normaloff>:/images/iconbulmacont.svg</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Nombre Empresa</string>
     </property>
     <property name="buddy">
      <cstring>mui_nomempresa</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="4">
    <widget class="QLineEdit" name="mui_nomempresa"/>
   </item>
   <item row="0" column="5" rowspan="5">
    <widget class="QToolButton" name="toolButton">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>15</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>..</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="6" rowspan="5">
    <widget class="QTextBrowser" name="mui_descripcion"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Nombre Base de Datos</string>
     </property>
     <property name="buddy">
      <cstring>mui_nomdb</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="3">
    <widget class="QLineEdit" name="mui_nomdb">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="4">
    <widget class="QCheckBox" name="mui_actualizarDB">
     <property name="text">
      <string>Actualizar Base de Datos</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Version de la Base de Datos</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="3">
    <widget class="QLineEdit" name="mui_databaserevision">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="5">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Opciones de Filtrado de Plugins</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="margin">
       <number>0</number>
      </property>
      <item>
       <widget class="QCheckBox" name="mui_vertodos">
        <property name="text">
         <string>Mostrar todos los plugins o solo los instalados</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>43</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Categoria </string>
          </property>
          <property name="buddy">
           <cstring>mui_categoria</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="mui_categoria"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="4" column="0" colspan="5">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="icon">
       <iconset resource="resources.qrc">
        <normaloff>:/images/iconbulmacont.svg</normaloff>:/images/iconbulmacont.svg</iconset>
      </attribute>
      <attribute name="title">
       <string>Plugins BulmaCont</string>
      </attribute>
      <layout class="QVBoxLayout">
       <property name="margin">
        <number>0</number>
       </property>
       <item>
        <widget class="QTableWidget" name="mui_plugins">
         <property name="alternatingRowColors">
          <bool>true</bool>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::NoSelection</enum>
         </property>
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
         <property name="gridStyle">
          <enum>Qt::NoPen</enum>
         </property>
         <column>
          <property name="text">
           <string>Plugin</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Descripcion</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Version Instalada</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_3">
      <attribute name="icon">
       <iconset resource="resources.qrc">
        <normaloff>:/images/terminal.png</normaloff>:/images/terminal.png</iconset>
      </attribute>
      <attribute name="title">
       <string>Consola</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QTextBrowser" name="mui_textBrowser"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QPushButton" name="mui_aceptar">
     <property name="text">
      <string>Aplicar cambios</string>
     </property>
     <property name="icon">
      <iconset resource="resources.qrc">
       <normaloff>:/images/bulmasetup_aceptar.xpm</normaloff>:/images/bulmasetup_aceptar.xpm</iconset>
     </property>
    </widget>
   </item>
   <item row="5" column="4">
    <widget class="QPushButton" name="mui_hacerbackup">
     <property name="text">
      <string>Hacer copia de seguridad</string>
     </property>
     <property name="icon">
      <iconset resource="resources.qrc">
       <normaloff>:/images/bulmasetup_save.png</normaloff>:/images/bulmasetup_save.png</iconset>
     </property>
    </widget>
   </item>
   <item row="5" column="5" colspan="2">
    <widget class="QPushButton" name="mui_cancelar">
     <property name="text">
      <string>Cerrar</string>
     </property>
     <property name="icon">
      <iconset resource="resources.qrc">
       <normaloff>:/images/bulmasetup_exit.png</normaloff>:/images/bulmasetup_exit.png</iconset>
     </property>
    </widget>
   </item>
   <item row="5" column="1" colspan="3">
    <widget class="QPushButton" name="mui_permisos">
     <property name="text">
      <string>Administrar Permisos</string>
     </property>
     <property name="icon">
      <iconset resource="resources.qrc">
       <normaloff>:/images/bulmasetup_editarusuario.xpm</normaloff>:/images/bulmasetup_editarusuario.xpm</iconset>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>mui_cancelar</sender>
   <signal>released()</signal>
   <receiver>ModificarContabilidadBase</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>790</x>
     <y>598</y>
    </hint>
    <hint type="destinationlabel">
     <x>297</x>
     <y>496</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>toolButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>mui_descripcion</receiver>
   <slot>setHidden(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>538</x>
     <y>198</y>
    </hint>
    <hint type="destinationlabel">
     <x>625</x>
     <y>270</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
